### DISCUSSION TITLE
WebRTC

### DISCUSSION THREAD

**Discussion by GSStnb** [Motion-Project/motionplus#190] [Enhancements]
I've been looking at a variety of streaming protocols to determine the best protocol for a centralized Motion/MotionPlus installation using network cameras (Raspberry Pi 0s).  I've looked at several applications and supported protocols (RTSP, RTMP and HTTP).  Each has pluses and minuses (bandwidth vs. quality vs ...).  I've also looked at options that are currently not supported by Motion/MotionPlus and one application I've come across is MediaMTX (formerly rtsp-simple-server). It supports the RTSP, etc, but in addition WebRTC.  WebRTC seems to offer a good quality video stream while not using a lot of bandwidth.  So, is  it possible to add WebRTC?

**Comment by Mr-Dave** [comment]
I would be interested in a pull request for this if you (or anyone else) wants to code this.

**Reply by GSStnb** [reply]
If someone else is interested, please don't hesitate.  It has been a very long time since I written any code.  I'll poke at it when I can, but that might be all I can do  :-)
